### PR TITLE
Auto-correct Rollbar.log calls.

### PR DESCRIPTION
**Pull request by cabello**
**Replace this text with a summary of the changes in your PR.
The more detailed you are, the better.**

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](https://github.com/wealthsimple/rubocop-vendor/blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](https://github.com/wealthsimple/rubocop-vendor/blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] Run `bundle exec rake default`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: spec/rubocop/cop/vendor/rollbar_logger_spec.rb [hunk 1]
```diff
@@ -38,11 +44,20 @@
...
     expect(autocorrect_source('Rollbar.debug("Stale message")')).to eq('Rails.logger.debug("Stale message")')
   end
 
-  it 'autocorrect `Rollbar.info` to `Rails.logger.info` ' do
+  it 'autocorrect `Rollbar.info <string>` to `Rails.logger.info`' do
     expect(autocorrect_source('Rollbar.info "Stale message"')).to eq('Rails.logger.info "Stale message"')
   end
+
+  it 'autocorrect `Rollbar.debug(<hash>)` to `Rails.logger.debug`' do
+    expect(autocorrect_source('Rollbar.debug({foo: "bar"})')).to eq('Rails.logger.debug({foo: "bar"})')
```
Comment (marcotc) [inline spec/rubocop/cop/vendor/rollbar_logger_spec.rb]: Could you check if this also works with `Rollbar.debug(foo: "bar")`? I believe it's a different AST.
If it doesn't and it's too hard to parse, we can leave it for now.

Comment (cabello) [reply spec/rubocop/cop/vendor/rollbar_logger_spec.rb]: Tested it and it works.
